neuro_extras/main.py

```diff
@@ -574,7 +563,37 @@ async def _transfer_image(source: str, destination: str) -> None:
                     """
                 )
             )
-        await _build_image("Dockerfile", tmpdir, destination, [], [], [])
+        return await _build_image("Dockerfile", tmpdir, destination, [], [], [])
+
+
+async def _attach_job_stdout(
+    job: neuro_api.JobDescription, client: neuro_api.Client, name: str = ""
+) -> int:
+    while job.status == neuro_api.JobStatus.PENDING:
+        job = await client.jobs.status(job.id)
+        await asyncio.sleep(1.0)
+    async for chunk in client.jobs.monitor(job.id):
+        if not chunk:
+            break
+        click.echo(chunk.decode(errors="ignore"), nl=False)
+    while job.status in (neuro_api.JobStatus.PENDING, neuro_api.JobStatus.RUNNING):
```

Comment:
mb lets put line above (_click.echo(chunk.decode(errors="ignore"), nl=False)_) into this cycle?

Reply:
sorry, I didn't get what you mean